Events.php

```diff
@@ -13,14 +13,13 @@
 use humhub\modules\announcements\widgets\CloseButton;
 use humhub\modules\announcements\widgets\ResetButton;
 use Yii;
-use yii\base\Object;
 
 /**
  * Description of Events
  *
  * @author davidborn
  */
-class Events extends Object
+class Events
```

Comment:
I'm not sure this is the right thing to do.
Humhub v1.3 has been updated to the latest version of Yii2 and fixes conflicts with PHP 7.2

Reply:
I'm using HH 1.3.0-beta.3 and this module don't works without this fix.

Reply:
Do you use the latest master-version or any of the release versions?
The latest release version is 0.2 but atm I'm building on v0.3 for HH 1.3

What kind of error message appears? Do you have the log for it? 
At my local test environment everything seems to be working with HH v1.3 beta 3 and this module..

Reply:
I have this issue with 0.2 or latest master with HH 1.3.0-beta.3.

```
[error][yii\base\ErrorException:64] yii\base\ErrorException: Cannot use yii\base\Object as Object because 'Object' is a special class name in /xxx/protected/modules/announcements/Events.php:15
```

With this fix, all works fine.

Reply:
Ah forgot that my local test environment didn't use PHP 7.2, so thats the reason I don't get any errors...
I guess you're right, in the [polls-module](https://github.com/humhub/humhub-modules-polls/blob/f557bb08d604ad46c1c5c2070640a89a2f22bfc4/Events.php#L23) it's solved in the same way

Thank you for your contributions